src/Types/Date.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Laudis Neo4j package.
+ *
+ * (c) Laudis technologies <http://laudis.tech>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Laudis\Neo4j\Types;
+
+use Bolt\structures\Date as BoltDate;
+
+class Date
+{
+    private int $days;
+
+    public function __construct(int $days)
+    {
+        $this->days = $days;
```

Comment:
I see this recurring theme in all the Temporal Classes. Are these all the variables returned by cypher? 

When looking at the docs: https://neo4j.com/docs/cypher-manual/current/functions/temporal/ there really seems to be a lot more variables. But then again, I only worked with timestamps, so I don't know.

Reply:
Indeed. Same comment about parsing from Bolt's structures. I'll have to dig into the response to get the rest. One other obvious transition here is a parent class (contemplated earlier but wanted a draft for review and then i enhance as we go).